### ISSUE TITLE
Vector layer added via Layman is not styled

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#3513]
# Feature Request

**Is your feature request related to a problem? Please describe.**
Related to #3307 
Adding a vector file (GeoJSON) with style file directly into a map is OK. Adding a vector file (GeoJSON) with style file while also publishing the layer into Layman results into a layer without styling.

**Describe the solution you'd like**
Also load layer's styles from the Layman.

**Describe alternatives you've considered**
none

**Additional context**
spin-off from #3307

**Comment by raitisbe** [comment]
@jmacura what is the style definition for that layer in layman in this case? Is it empty or could it be in SLD 1.1 format maybe?

**Comment by jmacura** [comment]
@raitisbe I have tested now with SLD 1.0 and the styles were loaded indeed... 🤔 
![image](https://user-images.githubusercontent.com/5598693/206301236-f9966b1a-4907-4fc3-9564-8024a0862ef9.png)

So, yes, the issue might be related to SLD 1.1 or QML.
@fzadrazil Do you have any specific use case when the style upload fails, which you could share with us?

**Comment by raitisbe** [comment]
@jmacura Please check if https://github.com/hslayers/hslayers-ng/pull/3616 solves it

**Comment by jmacura** [comment]
Problem is with QML styles.

**Comment by jmacura** [comment]
We already support QML styles (cf. #3435). Seems like the QML file structure has changed recently and it is not yet reflected by geostyler-qgis-parser.

**Comment by FilipLeitner** [comment]
Update: https://github.com/geostyler/geostyler-qgis-parser/pull/569

**Comment by fzadrazil** [comment]
Try to finish the referenced PR from Geostyler. If there is no response from its authors, create our own fork.

**Comment by jmacura** [comment]
needs to be tested with Layman v 1.23 and Layman 2.0
